src/app/modals/about-modal.component.html

```diff
@@ -0,0 +1,25 @@
+<div class="about-modal-pf">
+  <div class="modal-header">
+    <button type="button" class="close" (click)="close($event)" aria-hidden="true">
+      <span class="pficon pficon-close"></span>
+    </button>
+  </div>
+  <div class="modal-body">
+    <h1 *ngIf="config.title">{{config.title}}</h1>
+    <div *ngIf="config.productInfo && config.productInfo.length > 0" class="product-versions-pf">
+      <ul class="list-unstyled">
+        <li *ngFor="let info of config.productInfo">
+          <strong>{{info.name}}</strong> {{info.value}}</li>
+      </ul>
+    </div>
+    <div class="product-versions-pf">
+      <ng-template *ngIf="template" [ngTemplateOutlet]="template">
+      </ng-template>
```

Comment:
Lets use an ng-content tag here.

Reply:
Do we need a `select` attribute for ng-content?

example:
<ng-content [select]="modal-body"></ng-content>

Reply:
I don't believe so. That says to add content only if the "element has modal-body attribute".